yt/yt/server/http_proxy/coordinator.cpp

```diff
@@ -522,6 +522,36 @@ void THostsHandler::HandleRequest(
     }
 }
 
+
+////////////////////////////////////////////////////////////////////////////////
+
+TClusterConnectionHandler::TClusterConnectionHandler(NApi::IClientPtr client)
+    : Client_(std::move(client))
+{ }
+
+void TClusterConnectionHandler::HandleRequest(
+    const NHttp::IRequestPtr& req,
+    const NHttp::IResponseWriterPtr& rsp)
+{
+    const auto& path = req->GetUrl().Path;
+    if (path != "/cluster_connection" && path != "/cluster_connection/") {
+        rsp->SetStatus(EStatusCode::NotFound);
+        WaitFor(rsp->Close())
+            .ThrowOnError();
+        return;
+    }
+
+    TGetNodeOptions options;
+    options.ReadFrom = EMasterChannelKind::Cache;
```

Comment:
It is a little bit preferrable to write:
```
auto options = TGetNodeOptions{.ReadFrom = EMasterChannelKind::Cache};
```

Reply:
This code does not compile :( 

```
/home/ytsaurus/yt/yt/server/http_proxy/coordinator.cpp:544:37: error: field designator 'ReadFrom' does not refer to any field in type 'TGetNodeOptions'
    auto options = TGetNodeOptions{.ReadFrom = EMasterChannelKind::Cache};
```

Looks like some C++ limitation

We have
```
struct TGetNodeOptions : public TMasterReadOptions

struct TMasterReadOptions {
    EMasterChannelKind ReadFrom;
}
```

And only
```
TMasterReadOptions{.ReadFrom = EMasterChannelKind::Cache}
```
works. But not in this case